ext/fs/Cargo.toml

```diff
@@ -25,6 +25,7 @@ fs3.workspace = true
 libc.workspace = true
 log.workspace = true
 rand.workspace = true
+rayon = "1.8.0"
```

Comment:
Can't we use spawn_blocking instead. Bringing rayon for these APIs seems like a lot

Reply:
`spawn_blocking` cannot be used in non-async context. 

Looking at `Cargo.lock`, `rayon` only adds crossbeam dependency. `rayon` is a little more suitable here because we want to parallelize large amounts of small file copies.

Reply:
Okay, let's land as is